lib/internal/http2/core.js

```diff
@@ -1684,6 +1685,16 @@ class ClientHttp2Session extends Http2Session {
     if (options.waitForTrailers)
       stream[kState].flags |= STREAM_FLAGS_HAS_TRAILERS;
 
+    const signal = options.signal;
+    if (options.signal) {
```

Comment:
```suggestion
    if (signal) {
```

In case it's a nasty getter :D